holidays/countries/aland_islands.py

```
@@ -0,0 +1,113 @@
+        # Midsummer Eve.
+        name = tr("Midsommarafton")
+        dt = (
+            self._add_holiday_1st_fri_from_jun_19(name)
+            if self._year >= 1953
+            else self._add_holiday_jun_23(name)
+        )
+
+        # Midsummer Day.
+        self._add_holiday(tr("Midsommardagen"), _timedelta(dt, +1))
+
+      
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)